aten/src/ATen/native/native_functions.yaml

```diff
@@ -501,6 +501,9 @@
 - func: randperm_out(Tensor result, int64_t n, *, Generator* generator=nullptr) -> Tensor
   variants: function
 
+- func: randint(Type dtype, IntList size, int64_t low, int64_t high, *, Generator* generator=nullptr) -> Tensor
```

Comment:
This should follow the same argument ordering and conventions as numpy randint:

https://docs.scipy.org/doc/numpy/reference/generated/numpy.random.randint.html

Reply:
Oh okay!